pkg/static/login.js

```diff
@@ -563,6 +563,39 @@
         }
     }
 
+    function render_recent_hosts() {
+        const hosts = JSON.parse(localStorage.getItem("cockpit-client-sessions") || "[]");
```

Comment:
Is it possible that this variable gets corrupted and then `JSON.parse()` will throw, and prevent the login page from showing properly?

If not, then probably it makes sense to move the default `[]` outside of being parsed.

Reply:
I wrapped it into try/catch block to prevent breaking of the whole login page if this variable would be corrupted. Good catch!